lib/manager/npm/extract/pnpm.ts

```diff
@@ -0,0 +1,105 @@
+import yaml from 'js-yaml';
+import { logger } from '../../../logger';
+import {
+  findLocalSiblingOrParent,
+  getSiblingFileName,
+  localPathExists,
+  readLocalFile,
+} from '../../../util/fs';
+import type { PackageFile } from '../../types';
+import type { PnpmWorkspaceFile } from './types';
+import { matchesAnyPattern } from './utils';
+
+export async function extractPnpmFilters(
+  fileName: string
+): Promise<string[] | null> {
+  try {
+    const contents = yaml.safeLoad(await readLocalFile(fileName, 'utf8'), {
+      json: true,
+    }) as PnpmWorkspaceFile;
+    if (
+      !Array.isArray(contents.packages) ||
+      !contents.packages.every((item) => typeof item === 'string')
```

Comment:
Use `is.string` function.

Reply:
changed in 4eb038a3e1226d588d9fd5d05e9466104cd76bbd